### DISCUSSION TITLE
Problem with 0.9.483

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#1024] [Q&A] [answered]
After updating the latest Quarto my long report (see https://fharrell.com/post/rflow.html) no longer renders correctly.  There is no floating table of contents and htmlwidgets and tabs are not correct.  The html file is 1/4 smaller.  I downgraded to 0.9.462 and I again get the correct output that is posted on that web site.

**Comment by dragonstyle** [comment]
I suspect I know the issue here - I've been working on our extension support and in doing so (and in particular in the way we inject HTML dependencies), self contained isn't working properly. If I recall correctly from reading your article, you do tend to use self contained. 

I intend to fix this issue tomorrow AM, so the best bet until tomorrow would be to stick on `0.9.462` until I resolve that, if you're ok with that.

Sorry about that - @jjallaire flagged this for me Friday afternoon and I just didn't get to it!

**Reply by harrelfe** [reply]
Not a problem at all.  Thanks.  Yes I use `self-contained`.

**Comment by dragonstyle** [comment, marked as answer]
This should now be fixed as of https://github.com/quarto-dev/quarto-cli/releases/tag/v0.9.486 . Sorry about that!

If you have a moment to confirm that all is well, I would appreciate knowing that this resolved even very complex cases such as yours @harrelfe. Thank you!

**Reply by harrelfe** [reply]
It's all good.  Thanks Charles!

**Reply by dragonstyle** [reply]
Great thank you for confirming!
